app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class AuditMailer < ApplicationMailer
+  def failure(audit: nil, package:, error:)
+    @error = error
+    mail(to: [audit.user.email, package.user.email], subject: "Dark Blue Audit Failure for #{package.content_type} package #{package.external_id}")
```

Comment:
Minor: This method has too many responsibilities; it constructs the addressees, subject, and sends the message.